client/asset/btc/btc.go

```diff
@@ -284,25 +284,6 @@ func CommonConfigOpts(symbol string /* upper-case */, withApiFallback bool) []*a
 	return opts
 }
 
-// SPVConfigOpts are the options common to built-in SPV wallets.
-func SPVConfigOpts(symbol string) []*asset.ConfigOption {
-	return []*asset.ConfigOption{{
```

Comment:
Now existing users will all have to rescan from the default birthday. This feature was very helpful for me once when I has having a hard time getting the wallet to sync and I knew the wallet was completely empty before I deposited coins just a few days earlier.

Reply:
I think this is worth it to keep things simple, although I also was using this to get sync to work. New users won't need to do anything though after this change.

We shouldn't build around bugs though, and btc not syncing is some bug or bugs in neutrino I think. Hopefully they get fixed...